python/ray/tests/test_client.py

```diff
@@ -284,6 +284,18 @@ def f():
         assert ray.get(f.remote()) == "OK"
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="Failing on Windows.")
+def test_function_and_get_batch(ray_start_regular_shared):
+    with ray_start_client_server() as ray:
+
+        @ray.remote
+        def f():
+            return "OK"
+
+        assert ray.get(
```

Comment:
Hmm, could we break the abstraction barrier and look at the `ray.worker.dataclient.req_id` and see if it only gets incremented by one?

Reply:
Done.